src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -0,0 +1,1908 @@
+                    passage=passage,
+                    question=question,
+                    passage_prefix="Paragraf: ",
+                    question_prefix="Pertanyaan: ",
+                )
+                references = []
+                for answer in row["answers"]["text"]:
+                    output = Output(text=answer.strip())
+                    references.append(Reference(output, tags=[CORRECT_TAG]))
+                instance = Instance(
+                    input=input,
+                    references=references,
+                    split=self.splits[split]
+                )
+                outputs.append(instance)
+        return outputs
+
+# 1.2 Vietnamese & Thai: XQuAD
+class XQuADScenario(Scenario):
+    """
+    XQuAD is an open-book question answering scenario that is parallel across 10 languages.
+    The dataset consists of a subset of 240 paragraphs and 1190 question-answer pairs from the
+    development set of SQuAD v1.1 (Rajpurkar et al., 2016) together with their professional translations.
+
+    This scenario only uses the Vietnamese and Thai subsets of the data and there are no
+    unanswerable questions.
+
+    The models are prompted using the following general format:
+
+        You will be given a paragraph and a question. Answer the question by extracting the answer from the paragraph.
+
+        Paragraph: <text>
+        Question: <question>
+        Answer: <answer>
+
+        ...
+
+        Paragraph: <text>
+        Question: <question>
+        Answer:
+
+    Target completion:
+        <answer>
+
+    @article{Artetxe:etal:2019,
+      author    = {Mikel Artetxe and Sebastian Ruder and Dani Yogatama},
+      title     = {On the cross-lingual transferability of monolingual representations},
+      journal   = {CoRR},
+      volume    = {abs/1910.11856},
+      year      = {2019},
+      archivePrefix = {arXiv},
+      eprint    = {1910.11856}
+    }
+    """
+
+    name = "xquad"
+    description = "Vietnamese and Thai Open-book Question Answering task"
+    tags = ["question_answering"]
+
+    def __init__(self, language: str):
+        super().__init__()
+        self.language = language
+        self.splits = {
+            "train": TRAIN_SPLIT,
+            "test": TEST_SPLIT
+        }
+        self.map = {
+            "th": {
+                "passage_prefix": "ข้อความ: ",
+                "question_prefix": "คำถาม: ",
+                "random_state": 4520,
+            },
+            "vi": {
+                "passage_prefix": "Đoạn văn: ",
+                "question_prefix": "Câu hỏi: ",
+                "random_state": 4502,
+            }
+        }
+
+    def get_instances(self, output_path) -> List[Instance]:
+        dataset = datasets.load_dataset("xquad", f"xquad.{self.language}", split="validation")
+        df = dataset.to_pandas()
+
+        # Sample 100 examples for test
+        df_test = df.sample(n=100, random_state=self.map[self.language]["random_state"])
```

Comment:
likewise

Reply:
We can increase this to 1000 and still have 190 remaining for 5-shot examples, although we might then want to consider putting the ones with the shortest lengths in the 190.